lib/vehicle_factory.rb

```diff
@@ -0,0 +1,22 @@
+class VehicleFactory
+
+  def initialize
+  end
+
+  def create_vehicles(registrations)
+    vehicles = []
+
+    registrations.each do |registration|
+      vin = registration[:vin_1_10]
+      year = registration[:model_year]
+      make = registration[:make]
+      model = registration[:model]
+
+      new_vehicle = Vehicle.new({vin: vin, year: year, make: make, model: model, engine: :ev})
+
+      vehicles << new_vehicle
+    end
+    vehicles
+  end
+  
+end
```

Comment:
Awesome work here. No need for the `initialize` method though. We only need to create an `initialize` method if we want to set attributes upon initialization of an object. Since we don't need to store any state on a VehicleFactory object once it's created, we don't even need to include this method.